service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwin.java

```diff
@@ -504,11 +504,7 @@ private DeviceTwinDevice jsonToDeviceTwinDevice(String json) throws IOException
         deviceTwinDevice.setCapabilities(twinState.getCapabilities());
         deviceTwinDevice.setConnectionState(twinState.getConnectionState());
         deviceTwinDevice.setConfigurations(twinState.getConfigurations());
-
```

Comment:
Have you verified that this works for non PnP device before making the change?

Reply:
yes for non pnp this value is set to null, when printed it will look like below:

Device ID: <id>
ETag: <etag>
**Model ID:** 
Tags:{}
Reported Properties{}
Desired Properties: {}

Reply:
👍